modules/charon-core/src/main/java/org/wso2/charon3/core/objects/plainobjects/Cursor.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Update the license header. Refer the template in https://somindagamage.medium.com/adding-and-using-wso2-copyright-license-in-intellij-idea-8d172ce8c420